packages/react-vapor/src/components/button/examples/ButtonExamples.1.use.md

```diff
@@ -0,0 +1,39 @@
+## Guidelines
```

Comment:
So, as discussed, I've just tried to apply the doc review changes myself. I havent built the project to see how it renders though, and I've left two questions below. I also applied Olivier’s comments while I was at it. 

I’ve changed the bolding quite a lot, don’t hesitate if you dislike what I’ve done, I could change it back :)
I don't know what your bolding practices are in the UX team, and if you want to discuss I'm sure it would be beneficial to me.